srcpkgs/pdfarranger/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'pdfarranger'
+pkgname=pdfarranger
+version=1.9.2
+revision=1
+build_style=python3-module
+hostmakedepends="intltool python3-distutils-extra python3-setuptools"
+depends="img2pdf poppler-glib python3-dateutil python3-gobject python3-pikepdf"
+short_desc="Simple application for PDF Merging, Rearranging, and Splitting"
+maintainer="Eloi Torrents <eloitor@disroot.org>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/pdfarranger/pdfarranger"
+distfiles="https://github.com/pdfarranger/pdfarranger/archive/refs/tags/${version}.tar.gz"
+checksum=e0e08ddc3a323940d7f05cc0e996484775ad7a0a6cbfc197b07473c452c43f66
+# tests require involved xvfb setup
+make_check=no
```

Comment:
would `make_check_pre="xvfb-run"` work?

Reply:
I'm not sure, I tried it and added `xvfb-run` in the checkdepends but it didn't work:

```
=> pdfarranger-1.9.2_1: running do_check ...
============================= test session starts ==============================
platform linux -- Python 3.11.2, pytest-7.2.1, pluggy-1.0.0
rootdir: /builddir/pdfarranger-1.9.2
collected 0 items

============================ no tests ran in 0.03s =============================
=> ERROR: pdfarranger-1.9.2_1: do_check: 'PYTHONPATH="$(cd build/lib* && pwd)" ${make_check_pre} python3 -m pytest ${testjobs} ${make_check_args} ${make_check_target}' exited with 5
=> ERROR:   in do_check() at common/build-style/python3-module.sh:15
```

Reply:
This suggests there are no tests in the tarball. If that's true, please change the comment on 'make_check=no` to note that there are no tests in the source distribution.